src/demo-app/a11y/a11y.html

```diff
@@ -1,9 +1,12 @@
 <h1>Accessibility Demo</h1>
 
 <nav>
+  <a (click)="skipNavigation()" href="javascript:void(0)">Skip navigation</a>
```

Comment:
Can this be done with a url fragment? Using `javascript:void(0)` is a pattern that is generally considered harmful to a11y